src/libasr/pass/intrinsic_function_registry.h

```diff
@@ -2106,7 +2227,10 @@ namespace IntrinsicFunctionRegistry {
                 {"sum", {&Sum::create_Sum, &Sum::eval_Sum}},
                 {"list.index", {&ListIndex::create_ListIndex, &ListIndex::eval_list_index}},
                 {"list.reverse", {&ListReverse::create_ListReverse, &ListReverse::eval_list_reverse}},
-                {"list.pop", {&ListPop::create_ListPop, &ListPop::eval_list_pop}}
+                {"Symbol", {&SymbolicSymbol::create_SymbolicSymbol, &SymbolicSymbol::eval_SymbolicSymbol}},
+                {"list.pop", {&ListPop::create_ListPop, &ListPop::eval_list_pop}},
+                {"SymbolicAdd", {&SymbolicAdd::create_SymbolicAdd, &SymbolicAdd::eval_SymbolicAdd}},
+                {"pi", {&SymbolicPi::create_SymbolicPi, &SymbolicPi::eval_SymbolicPi}},
```

Comment:
I would put `list.pop` at the top, right now it's between `Symbol` and `SymbolicAdd`.